client/pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>software.amazon.awssdk.iotdevicesdk</groupId>
             <artifactId>aws-iot-device-sdk</artifactId>
-            <version>1.17.0-CLUSTER-SNAPSHOT</version>
```

Comment:
Looks like CLI in development still needs cluster support. Could you create the release branch and cut a PR to release branch directly?